### DISCUSSION TITLE
Project structure

### DISCUSSION THREAD

**Discussion by romanzdk** [sqlalchemy/alembic#1444] [Usage Questions]
Is there any recommended way how to structure alembic project? Should I have something like
```
- project_name
-- .gitignore
-- Makefile
-- alembic_src
--- migrations
--- alembic.ini
--- ...
```

or e.g.
```
- project_name
-- alembic.ini
-- migrations
-- .gitignore
-- Makefile
-- ...
```

or even
```
- project_name
-- .gitignore
-- Makefile
-- alembic.ini
-- alembic_src
--- migrations
--- ...
```
?

**Comment by CaselIT** [comment]
Hi,

That mainly depends on where you plan to run the alembic command.

By default alembic will look for its alembic.ini file in the folder you launch it's command.

Probably the second one one it the most common
